firebase_admin/auth.py

```diff
@@ -109,10 +109,13 @@ def create_custom_token(uid, developer_claims=None, app=None):
 
     Raises:
         ValueError: If input parameters are invalid.
+        AuthError: If an error occurs while creating the token using the remote IAM service.
     """
     token_generator = _get_auth_service(app).token_generator
-    return token_generator.create_custom_token(uid, developer_claims)
-
+    try:
+        return token_generator.create_custom_token(uid, developer_claims)
```

Comment:
It looks like we are wrapping some calls with try catch (I assume because we now no longer assume that IAM calls work).  How confident are you that you caught all the cases.  This feels like it would be easy to miss one of these cases.